providers/openfeature-flagd-provider/lib/openfeature/flagd/provider/version.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module OpenFeature
+  module FlagD
+    VERSION = "0.0.1"
+  end
+end
```

Comment:
No idea if this is necessary or valid. It was the pattern we used in the Ruby SDK and it looks like Release Please looks for a `version.rb` file.